src/write_organ_tags.py

```diff
@@ -6,9 +6,77 @@
 #  Output will be used to filter and/or boost search results in Cell Annotation Platform (CAP)
 
 import argparse
-import sys
 import ruamel.yaml
 from SPARQLWrapper import SPARQLWrapper, JSON
+from collections import namedtuple
+
+
+def run_query(query):
+    sparql.setQuery(query)
+    ret = sparql.queryAndConvert()
+    output = []
+    for line in ret["results"]["bindings"]:
+        output.append(line)
+    return output
+
+
+def generate_organ_cells(output):
+    organs = []
+    for n in output:
+        curie = n['x']['value'].replace("_", ":")
+        curie = curie.partition('http://purl.obolibrary.org/obo/')[-1]
+        dl_query = "CL:0000000 and BFO:0000050 some " + curie
```

Comment:
As an example of a place whereby I could use a comment, here is where an explanation is necessary:

`dl_query = "CL:0000000 and BFO:0000050 some " + curie`

Why both CL and BFO?

Reply:
I think this is trying to say things that are 'Cell', CL:0000000, and 'part of', BFO:0000050, some specific organs. Make sense?

Reply:
It is good practice to write out queries using labels in comments.